roxie/ccd/ccdstate.cpp

```diff
@@ -453,6 +455,8 @@ class CRoxiePackageNode : extends CPackageNode, implements IRoxiePackage
     // Use dali to resolve subfile into physical file info
     static IResolvedFile *resolveLFNusingDaliOrLocal(const char *fileName, bool useCache, bool cacheResult, bool writeAccess, bool alwaysCreate, bool resolveLocal, bool isPrivilegedUser)
     {
+        unsigned hash = hashc((const unsigned char *) fileName, strlen(fileName), 0);
```

Comment:
minor comment.  0x811C9DC5 is normally used as the hash seed.  I suspect it doesn't matter here.